tests/integration/high_availability/test_self_healing.py

```diff
@@ -45,6 +44,7 @@
 METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
 APP_NAME = METADATA["name"]
 MYSQL_DAEMON = "mysqld"
+WAIT_TIMEOUT = 30 * 60
```

Comment:
It might be too much... but let's try with 30 minutes while we are on overloaded GH runners.